tests/playwright/Report.md

```diff
@@ -0,0 +1,213 @@
+# Evaluation of Microsoft Playwright
+
+## Table of Contents
+
+[Introduction](#Introduction)
+
+[Project Background](#project-background)
+
+[Playwright Vs WebdriverIO](#playwright-vs-webdriverIO)
+
+* [Sample Playwright Page screenshot code](#sample-playwright-page-screenshot-code)
+* [Playwright and Jest](#playwright-and-jest)
+  * [Example of jest-playwright.config.js](#example-of-jest-playwright-config-js)
+* [Playwright and WebdriverIO metrics](#playwright-and-webdriverIO-metrics)
+  * [Statistics on WebdriverIO and Playwright Popularity](#statistics-on-webdriverIO-and-playwright-popularity)
```

Comment:
In the spirit of simple titles, I would suggest renaming this to just `Statistics`.